### DISCUSSION TITLE
小小吐槽一下

### DISCUSSION THREAD

**Discussion by TNCNET** [share121/inter-knot#5222] [灌水]
嗯，这么说吧
之前不是举报了一个二维码吗？
毕竟自己的话，下意识是认为这有nsfw内容的，嗯，毕竟之前还看过一些真的开车（？）的帖子。

结果就发现是被rickroll了（
算了，下次统一拿扫码器确认一下，避免再上钩。

**Comment by ParanoidHikari** [comment]
你也被钓鱼啦（
虽然但是，扫码建议做好保护措施，万一有病毒就惨了

**Reply by TNCNET** [reply]
还真是，虽然我一般来说会先使用备用机扫一次（

**Comment by fzdqz** [comment]
盲猜是被我钓上来的∠( ᐛ 」∠)_

**Reply by TNCNET** [reply]
看了一下，还真是（悲
